ts/features/newWallet/screens/WalletHomeScreen.tsx

```diff
@@ -19,17 +20,34 @@ const WalletHomeScreen = () => {
     dispatch(idPayWalletGet.request());
   }, [dispatch]);
 
+  if (cards.length === 0) {
+    return (
+      <ScrollView contentContainerStyle={IOStyles.flex}>
+        <WalletPaymentsRedirectBanner />
```

Comment:
There's an issue with the banner which doesn't respect the horizontal margins: 

![simulator_screenshot_81AC23D0-5014-417D-83DC-B69E9CFB2448](https://github.com/pagopa/io-app/assets/12371664/1124c3d9-5bb5-4308-8554-6b4745400fdf)

Reply:
Fixed in https://github.com/pagopa/io-app/pull/5645/commits/9a86bd22eff4f65caad10d037ce6f136201daa39, lmk!